### ISSUE TITLE
Объясните мне почему такая большая комиссия?

### ISSUE THREAD

**Issue by VitalyZagorulko** [bitpay/copay#7474]
Я хотел отправить на localbitcoins 0.001 биткоин, в итоге написано что отправлено 0.00005 биткоина и комиссия 0.000949. Как это понимать объясните пожалуйста

**Comment by Marabun** [comment]
Да, такие в данный момент комиссии. Высокая нагрузка на сеть. При этом Copay ставит даже немного выше, чем средняя. Вообще советую переходить на кошельки поддерживающие segwit, в этом случае комиссия будет приблизительно в два раза меньше. Сopay, к сожалению, не поддерживает segwit.
